lib/src/generators/typescript/__snapshots__/axios-client.spec.ts.snap

```diff
@@ -201,6 +240,45 @@ export async function createUser(request: CreateUserRequest, authToken: string |
     }
 }
 
+export async function getUsers(): Promise<{
```

Comment:
It looks like the parameter is missing in the generated TypeScript library.

Reply:
true, I forgot to add it to the axios client